prac_05/emails.py

```diff
@@ -0,0 +1,14 @@
+email_to_name = {}
+email = input("Email: ")
+while email != "":
+    modified_email = email.replace(",", " ").replace(".", " ")
+    name = " ".join([char for char in (modified_email[:modified_email.find("@")].split()) if not char.isdigit()]).title()
```

Comment:
This is not very readable. It's not usually better to write hard-to-read code on one line.

Reply:
I have moved this section in to a get_names_from_email function and changed the code to be more readable.